libraries/AP_TECS/AP_TECS.h

```diff
@@ -429,6 +431,10 @@ class AP_TECS {
 
     // need to reset on next loop
     bool _need_reset;
+    // Flag if someone else drives pitch externally.
+    bool _flag_pitch_forced;
```

Comment:
i'm not keen on the reset code as a special case for under 1 degree pitch range, it implies we have some bad behaviour at narrow pitch ranges, eg. 1.5 degrees. If we don't have any bad behaviour then why do we need the special case?